pennylane/devices/qubit/simulate.py

```diff
@@ -52,16 +53,15 @@
     "jax": "jax",
     "torch": "torch",
     "tensorflow": "tensorflow",
-    # map non-standard interfaces to those known by autoray
-    "auto": None,
-    "scipy": "numpy",
-    "jax-jit": "jax",
-    "jax-python": "jax",
-    "JAX": "jax",
-    "pytorch": "torch",
-    "tf": "tensorflow",
-    "tensorflow-autograph": "tensorflow",
-    "tf-autograph": "tensorflow",
+    # map canonical interfaces
```

Comment:
I wonder if we can eliminate our need for these maps?

Reply:
I don't think we can completely eliminate the need for mapping to autoray strings.

Autoray currently relies on specific strings for interfaces that conflict with our internal logic. For example, it treats 'tf' and 'tf-autograph' the same both being equivalent to the 'tensorflow' autoray interface. This is why the recent commits add an `Interface.get_like()` method allowing us to provide the 'like=X' format that Autoray expects in files like `simulate.py`.